_posts/news/2021-07-01-IROS-2021.md

```diff
@@ -6,4 +6,15 @@ author: Alessandro Roncone
 ---
 
 Congratulations to [Caleb]({% post_url people/2019-10-10-caleb %}), Kandai, Matt, Mary, Ander and Krishna for their paper acceptance at IROS 2021!
+
+
+#### _Contact Anticipation for Physical Human-Robot Interaction with Robotic Manipulators using Onboard Proximity Sensors_ by [Caleb Escobedo]({% post_url people/2019-10-10-caleb %}), <a href="https://www.linkedin.com/in/matthewhstrong/">Matthew Strong</a>, Mary West, Ander Aramburu, and Alessandro Roncone
+
+This publication introduces dynamic contact thresholding, a method for contact anticipation during physical human-robot interaction in close proximity dynamic environments. Additionally, a framework is introduced that outlines object detection, prior to contact behavior, contact detection, and post contact behavior for a robot manipulator.
```

Comment:
I feel that you should introduce the framework, then say how dynamic contact thresholding is the core element of it.

Reply:
I think that would make it seem like the framework is more important than the dynamic contact thresholding. I'm thinking of a way to change the wording.

Reply:
Ah, sounds good